server/server/session/src/main/java/com/alipay/sofa/registry/server/session/push/PushEfficiencyImproveConfig.java

```diff
@@ -0,0 +1,267 @@
+   * @return
+   */
+  public boolean inAppSBF(String appName) {
+    if (CollectionUtils.isNotEmpty(subAppSet) && subAppSet.contains(appName)) {
+      return true;
+    }
+    return false;
+  }
+
+  public PushEfficiencyImproveConfig() {}
+
+  public int getChangeDebouncingMillis() {
+    if (inIpZoneSBF()) {
+      return changeDebouncingMillis;
+    }
+    return DEFAULT_CHANGE_DEBOUNCING_MILLIS;
+  }
+
+  public void setChangeDebouncingMillis(int changeDebouncingMillis) {
+    this.changeDebouncingMillis = changeDebouncingMillis;
+  }
+
+  public int getChangeDebouncingMaxMillis() {
+    if (inIpZoneSBF()) {
+      return changeDebouncingMaxMillis;
+    }
+    return DEFAULT_CHANGE_DEBOUNCING_MAX_MILLIS;
+  }
+
+  public void setChangeDebouncingMaxMillis(int changeDebouncingMaxMillis) {
+    this.changeDebouncingMaxMillis = changeDebouncingMaxMillis;
+  }
+
+  public int getChangeTaskWaitingMillis() {
+    if (inIpZoneSBF()) {
+      return changeTaskWaitingMillis;
+    }
+    return DEFAULT_CHANGE_TASK_WAITING_MILLIS;
+  }
+
+  public void setChangeTaskWaitingMillis(int changeTaskWaitingMillis) {
+    this.changeTaskWaitingMillis = changeTaskWaitingMillis;
+  }
+
+  public int getPushTaskWaitingMillis() {
+    if (inIpZoneSBF()) {
+      return pushTaskWaitingMillis;
+    }
+    return DEFAULT_PUSH_TASK_WAITING_MILLIS;
+  }
+
+  public void setPushTaskWaitingMillis(int pushTaskWaitingMillis) {
+    this.pushTaskWaitingMillis = pushTaskWaitingMillis;
+  }
+
+  public int getPushTaskDebouncingMillis() {
+    if (inIpZoneSBF()) {
+      return pushTaskDebouncingMillis;
+    }
+    return DEFAULT_PUSH_TASK_DEBOUNCING_MILLIS;
+  }
+
+  public void setPushTaskDebouncingMillis(int pushTaskDebouncingMillis) {
+    this.pushTaskDebouncingMillis = pushTaskDebouncingMillis;
+  }
+
+  public int getRegWorkWaitingMillis() {
+    if (inIpZoneSBF()) {
+      return regWorkWaitingMillis;
+    }
+    return DEFAULT_REG_WORK_WAITING_MILLIS;
+  }
+
+  public void setRegWorkWaitingMillis(int regWorkWaitingMillis) {
+    this.regWorkWaitingMillis = regWorkWaitingMillis;
+  }
+
+  public Set<String> getIpSet() {
+    return ipSet;
+  }
+
+  public void setIpSet(Set<String> ipSet) {
+    this.ipSet = ipSet;
+  }
+
+  public Set<String> getZoneSet() {
+    return zoneSet;
+  }
+
+  public void setZoneSet(Set<String> zoneSet) {
+    this.zoneSet = zoneSet;
+  }
+
+  public Set<String> getSubAppSet() {
+    return subAppSet;
+  }
+
+  public void setSubAppSet(Set<String> subAppSet) {
+    this.subAppSet = subAppSet;
+  }
+
+  public int getSbfAppPushTaskDebouncingMillis() {
+    return sbfAppPushTaskDebouncingMillis;
+  }
+
+  public void setSbfAppPushTaskDebouncingMillis(int sbfAppPushTaskDebouncingMillis) {
+    this.sbfAppPushTaskDebouncingMillis = sbfAppPushTaskDebouncingMillis;
+  }
+
+  public void setSessionServerConfig(SessionServerConfig sessionServerConfig) {
+    if (null != sessionServerConfig && sessionServerConfig.getSessionServerRegion() != null) {
```

Comment:
use StringUtil.isBlank instead of !=null

Reply:
adjusted